### PR TITLE
done the challenges of the first exercise

### PR DESCRIPTION
**Pull request by Yjohn**

### CODE REVIEW

File: homework/advanced/challenge11.js [hunk 1]
```diff
@@ -15,4 +15,54 @@ e.g you can add interest to all of your accounts.
...
+        var average = 0;
+        for (var i = 0; i < this.acounts.length; i++) {
+           average += this.acounts[i].balance;
+        }
+        return average/this.acounts.length
+    },
+    Interest: function () {
+        var interest = 0;
+        for (var i = 0; i < this.acounts.length; i++) {
+           this.acounts[i].balance = this.acounts[i].balance*1.1;
```
Comment (NateWr) [inline homework/advanced/challenge11.js]: `acounts` should be `accounts`

**Comment by Yjohn** [comment]
i tried to fix the challenge depends on the comment that i received from Nate

File: homework/challenge04.js [hunk 2]
```diff
@@ -8,3 +8,25 @@ shortcut("goodbye")  // --> "gdby"
...
+    }
+    console.log(result);
+}
+var given_string = "My Name is YOHANNES!";
+removevowel(given_string);
+var given_string = ('goodbye');
+removevowel(given_string);
+var given_string = ('node girls');
+removevowel(given_string);
+var given_string = ('how are you today?');
```
Comment (NateWr) [inline homework/challenge04.js]: Don't forget to remove all of the `(` and `)` characters here. You don't need them for strings, so remove them from lines 27, 29 and 31.

File: homework/challenge05.js [hunk 3]
```diff
@@ -10,4 +10,49 @@ stringy(5) // --> '10101'
...
+        }
+        return result;
+    }
+    else {
+        console.log();
+    }
+}
+
+stringy(10);
+console.log(result);
```
Comment (NateWr) [inline homework/challenge05.js]: Good, but remember you can use the result of the `stringy` function directly. So you can change these two lines:

```js
stringy(10);
console.log(result);
```

To this:

```js
console.log(stringy(10));
```

**Comment by NateWr** [comment]
:100: 

Normally, this is when your pull request would be merged. But since this is just an exercise, I'll close this. Nice work!

**Comment by Yjohn** [comment]
Thanks for your all comments and help

**Pull request closed without merging**